tests/test_middleware.py

```diff
@@ -2,7 +2,7 @@
 from django.http import HttpResponse
 
 import mock
-from mohawk.exc import MacMismatch
+from mohawk.exc import MisComputedContentHash
```

Comment:
Oh, right, I see your point. The real fix would be to override this dep in `tox.ini` but that creates other problems. OK, let's leave 0.3.3 in setup.py even though it may not be entirely accurate. No one should be using an older version of mohawk anyway due to security bugs.